genapkovl-lima.sh

```diff
@@ -222,6 +235,38 @@ if [ "${LIMA_INSTALL_NERDCTL}" == "true" ]; then
     done
```

Comment:
might want to make this CNI installation optional, since it is not needed for nerdctl or buildkit (but by containerd)
Currrently it is a bit duplicated, which means that some of the CNI plugins will be installed twice - adding to size.

![alpine-libexec](https://user-images.githubusercontent.com/10364051/148207494-1adc673a-88b1-471c-bc2e-02161e678c4b.png)
I think that lima will install these (with containerd), as well ?

Reply:
https://github.com/lima-vm/lima/blob/v0.8.0/pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh#L12

Reply:
I removed the duplicates, of all the standard CNI plugins (but not https://github.com/AkihiroSuda/cni-isolation)